interop-testing/src/main/java/io/grpc/testing/integration/XdsTestServer.java

```diff
@@ -216,8 +233,33 @@ private void start() throws Exception {
               .build();
       server.start();
     } else {
+      ServerBuilder<?> serverBuilder;
+      ServerCredentials insecureServerCreds = InsecureServerCredentials.create();
+      switch (addressType) {
```

Comment:
Add a failure in the `if (secureMode)` block to fail if `addressType != IPV4_IPV6`?